subprojects/docs/src/docs/userguide/running-builds/configuration_cache.adoc

```diff
@@ -68,21 +75,27 @@ Apart from skipping the configuration phase, the configuration cache provides so
 It is recommended to get started with the simplest task invocation possible.
 Running `help` with the configuration cache enabled is a good first step:
 
-[source,text]
 ----
 ❯ gradle --configuration-cache help
 include::{snippetsPath}/configurationCache/noProblem/tests/store.out[]
+...
```

Comment:
Indeed. Most of the output is taken from the tests, line above your comment.

Outputs from the tests shows a build time of `0s`, this is changed only in this introduction to better illustrate the benefits. The test ensures that the build is successful. So I think we're fine here.

Then there's the post-build output. The tests don't capture this so it can't be taken from the test outputs :/